src/main/java/io/xpring/ilp/grpc/GrpcConstants.java

```diff
@@ -0,0 +1,7 @@
+package io.xpring.ilp.grpc;
+
+import io.grpc.Context;
+
+public final class GrpcConstants {
```

Comment:
Does this really need to be public? Could it be private or protected?

Reply:
We actually don't need this class at all.  It's only used on the server